lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -305,16 +342,38 @@ class CategoryRenderer {
 
     const metricAudits = category.audits.filter(audit => audit.group === 'perf-metric');
     const metricAuditsEl = this._renderAuditGroup(groups['perf-metric']);
+    const timelineContainerEl = this._dom.createChildOf(metricAuditsEl, 'div',
+        'lh-timeline-container');
+    const timelineEl = this._dom.createChildOf(timelineContainerEl, 'div', 'lh-timeline');
+
+    let perfTimelineScale = 0;
+    metricAudits.forEach(audit => {
+      if (Number.isFinite(/** @type {number} */ (audit.result.rawValue))) {
+        perfTimelineScale = Math.max(perfTimelineScale, audit.result.rawValue);
+      }
+    });
 
     const thumbnailAudit = category.audits.find(audit => audit.id === 'screenshot-thumbnails');
-    const thumbnailDetails = thumbnailAudit && thumbnailAudit.result &&
-        thumbnailAudit.result.details;
-    if (thumbnailDetails) {
+    const thumbnailResult = thumbnailAudit && thumbnailAudit.result;
+    if (thumbnailResult && thumbnailResult.details) {
+      const thumbnailDetails = /** @type {!DetailsRenderer.FilmstripDetails} */
+          (thumbnailResult.details);
+      if (typeof thumbnailDetails.scale === 'number') {
```

Comment:
on versions of the LHR generated before this PR, not sure if we care about breaking day-to-day, i'm fine removing